### PR TITLE
Hi!

### PR DESCRIPTION
**Pull request by PaltineanuRares956**

### CODE REVIEW

File: Paltineanu Rares-Mihai/3.7 - curs4/main.c [hunk 1]
```diff
@@ -1,48 +1,71 @@
...
+}
+
 int main()
 {
     char hex[50];
     double dec = 0;
     int i;
     gets(hex);
-    puts(hex);
+    while(verif(hex) == 0)
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/3.7 - curs4/main.c:31]: Look at you go! Great work

File: Paltineanu Rares-Mihai/3.7 - curs4/main.c [hunk 2]
```diff
@@ -1,48 +1,71 @@
...
     while(i >= 0)
     {
         if(hex[i] >= '0' && hex[i] <= '9')
-            dec = (hex[i]-'0')*pow(16,i);
+        {
+            dec = dec + (hex[i]-'0')*pow(16,strlen(hex) - i - 1);
+        }
         else
         {
             if(hex[i] == 'A')
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/3.7 - curs4/main.c:46]: Here's an instance where a `switch` might work as well!
Or maybe, one could extract all that in a function and simply have different parameters? Much of the code seems the same

File: Paltineanu Rares-Mihai/3.8 - curs4/main.c [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    for(int i = 0; i < 200; i++)
+    {
+        pol[i] = 0;
+    }
+
+    printf("Read the degree of the polynomial pol: ");
+    scanf("%d", &n);
+    printf("Read the value of x: ");
+    scanf("%d", &x);
+    for(int i = 0; i <= n; i++)
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/3.8 - curs4/main.c:20]: Again, nice work

File: Paltineanu Rares-Mihai/3.9 - curs 4/main.c [hunk 4]
```diff
@@ -0,0 +1,257 @@
+/**Write a program to perform the operations +,-,*, /  on two polynomials:
+A(x)=a0+a1x1+...+ anxn
+B(x)=b0+b1x1+......+ bmxm
+The degrees and the coefficients are read from the keyboard
+*/
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
```
Comment (Alex-Cosma) [inline Paltineanu Rares-Mihai/3.9 - curs 4/main.c:9]: Really long class with way too much stuff going on, but I haven't taught you guys functions yet and modular programming so it's totally reasonable.

Great job!

**Review by Alex-Cosma — APPROVED** [review]

File: Paltineanu Rares-Mihai/3.9 - curs 4/main.c (hunk 4 above)
Comment (PaltineanuRares956) [reply Paltineanu Rares-Mihai/3.9 - curs 4/main.c:9]: I just found out that i can respond to your comments. I would like to use functions, but i thought that it is better to code only in main until you will teach us functions next week. I will work with functions if it is ok.

Comment (Alex-Cosma) [reply Paltineanu Rares-Mihai/3.9 - curs 4/main.c:9]: Sure it is

**Pull request merged**